docs/development/doc_guide.rst

```diff
@@ -1,76 +1,829 @@
-************************
-Documentation Guidelines
-************************
+*********************
+Writing Documentation
+*********************
 
-This document describes the documentation requirements and guidelines
-to be followed during the development of PlasmaPy and affiliated
-packages.
+Documentation that is up-to-date and understandable is vital to the
+health of a software project. This page describes the documentation
+requirements and guidelines to be followed during the development of
+PlasmaPy and affiliated packages.
+
+.. hint::
+
+   Updating documentation is one of the best ways to make a first
+   contribution to an open source software project.
+
+.. note::
+
+   If you discover areas within PlasmaPy's documentation that are
+   confusing or incomplete, please `raise an issue`_! This really helps
+   PlasmaPy not only by helping us improve the documentation for all,
+   but also by creating opportunities for new contributors to make their
+   first contribution to the project.
+
+`PlasmaPy's documentation`_ is hosted by `Read the Docs`_ and is
+available at these locations:
+
+* The documentation corresponding to the most recent official release is
+  labelled ``stable`` and is found at `https://docs.plasmapy.org/`_ or
+  `https://docs.plasmapy.org/en/stable/`_.
+
+* The documentation corresponding to the ``main`` branch on `PlasmaPy's
+  GitHub repository`_ is labelled ``latest`` and can be found at
+  `https://docs.plasmapy.org/en/latest/`_.
```

Comment:
```suggestion
* The documentation corresponding to the most recent release to `PyPI`_ is
  labelled ``stable`` and is found at `https://docs.plasmapy.org/`_ or
  `https://docs.plasmapy.org/en/stable/`_.

* The documentation corresponding to the ongoing development on the
  ``main`` branch in `PlasmaPy's GitHub repository`_ , which is often ahead
  of the most recent release, is labelled ``latest`` and can be found at
  `https://docs.plasmapy.org/en/latest/`_.
```

I reworded this a little because I thought it would be useful to the reader to add a little more clarification about the difference between the most recent release and `main`.